src/main/java/io/ballerina/plugins/idea/configuration/ui/BallerinaSdkPanel.java

```diff
@@ -156,17 +156,17 @@ private void handleInitialCheckbox() {
                 } else {
                     selectedVersionTextField.setText(NO_VALID_BAL_SDK);
                     selectedVersionTextField.setForeground(JBColor.RED);
-                    selectedSdkVersion = EMPTY;
-                    selectedSdkPath = EMPTY;
+                    selectedSdkVersion = EMPTY_STRING;
+                    selectedSdkPath = EMPTY_STRING;
                 }
             }
         }
     }
 
     private void fillComboBox() {
-        for (BallerinaSdkUtils.BallerinaSdk sdk : sdkList) {
-            if (!Objects.equals(selectedSdkPath, sdk.getPath()) && !Objects.equals(systemBalPath, sdk.getPath())) {
-                sdkVersionComboBox.addItem(BallerinaSdkUtils.findBalDistFolder(sdk.getPath()));
+        for (BallerinaSdkUtils.ballerinaSdk sdk : sdkList) {
```

Comment:
Instead of doing this, we can move the `BallerinaSdk ` internal record to a separate class